susePublicCloudInfoClient/setup.py

```diff
@@ -9,17 +9,18 @@
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation, either version 3 of the License, or
 # (at your option) any later version.
-# 
+#
 # ec2uploadimg is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
-# 
+#
 # You should have received a copy of the GNU General Public License
 # along with ec2uploadimg. If not, see <http://www.gnu.org/licenses/>.
 
 import os
 import sys
+import version
```

Comment:
What about an explicit execution of the version module?

```
this_path = os.path.dirname(os.path.abspath(__file__))
version_path = os.path.join(this_path, 'lib', 'susepubliccloudinfoclient', 'version.py')
execfile(version_path)

# In setup ...
version=VERSION
```

That way the python path is not being messed with, the join will use proper os.sep and flake8 remains happy.